snap/snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
-name: jenkins
-title: Jenkins
+name: jenkins-lts
+title: Jenkins-lts
```

Comment:
Was this intentional? Surely you just want to keep this as `jenkins` and release to a different track so the experience is:

```bash
# "Normal" jenkins
snap install jenkins

# "LTS" jenkins
snap install jenkins --channel lts/stable
```

Reply:
Oh you know, I hadn't considered that...

It was intentional, but I see your point.